lib/sdk.js

```diff
@@ -284,35 +284,52 @@ Vantiq.prototype.query = function(source, params) {
 //  sources: Source events.  Path is the source name
 //  types:   Data Type events.  Path is the name and operation (e.g. "/MyType/insert")
 // 
-Vantiq.prototype.subscribe = function(resource, name, operation, callback) {
+Vantiq.prototype.subscribe = function(resource, name, parameters, callback) {
```

Comment:
I don't think we can change a parameter like this in the SDK without causing issues when users update to a new version of the SDK. I'm also concerned about overloading this parameter so sometimes it can be a string for the op, and sometimes it's an object for topic subscriptions. It would better to create a separate method with an extra property like you did in the java sdk for this.